internal/cli/inputs_test.go

```diff
@@ -218,14 +226,24 @@ func TestResolveGroupID(t *testing.T) {
 			},
 			expectedGroupID: testGroup.ID,
 		},
+		{
```

Comment:
can we also write a test that proves the default selection is present when auto confirm is not true?